docs/server/rq.rst

```diff
@@ -103,3 +143,32 @@ that all else is good and that the stats are fine or to be generated again.
 .. code-block:: bash
 
    $ redis-cli -n 2 del pootle:refresh:stats
+
+
+Delete dirty counts
++++++++++++++++++++
+
+Sometimes statistics are correctly calculated, but the banner telling that
+stats are being refreshed doesn't dissappear. This usually happens because some
+job failed and didn't decrease the dirty counts.
```

Comment:
From my experience this is due to an error in human intervention, e.g. workers were killed while there were ongoing jobs.